test/SlowTests/Issues/RavenDB-19100.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
+using FastTests;
+using FastTests.Client;
+using NuGet.ContentModel;
+using Orders;
+using Raven.Client;
+using Raven.Client.Documents;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Documents.Indexes;
+using Raven.Client.Documents.Operations;
+using Raven.Client.Documents.Operations.Indexes;
+using Raven.Client.Documents.Session;
+using Raven.Client.Exceptions.Database;
+using Raven.Client.Exceptions.Documents.Indexes;
+using Raven.Client.ServerWide;
+using Raven.Client.ServerWide.Operations;
+using Raven.Server;
+using Raven.Server.Commercial;
+using Raven.Server.Documents;
+using Raven.Server.Documents.Indexes;
+using Tests.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+using Xunit.Sdk;
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_19100 : ClusterTestBase
+    {
+        public RavenDB_19100(ITestOutputHelper output) : base(output)
+        {
+            DoNotReuseServer();
+        }
+
+        [Fact]
+        public async Task UpdateDbTopologyOperation_RemoveMember()
+        {
+            var (nodes, leader) = await CreateRaftCluster(3);
+            var storeOptions = new Options { Server = leader, ReplicationFactor = nodes.Count };
+
+            using (var store = GetDocumentStore(storeOptions))
+            {
+                var databaseName = store.Database;
+                var record = await store.Maintenance.Server.SendAsync(new GetDatabaseRecordOperation(databaseName));
+                var databaseTopology = new DatabaseTopology();
+                record.Topology.CopyTo(databaseTopology);
```

Comment:
I think you can modify the topology here directly

Reply:
Done